src/ontology/edits/oeo-social.omn

```diff
@@ -411,8 +411,9 @@ Class: OEO_00000341
 Class: OEO_00000367
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A sector is a subdivision in an economic system."@en,
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/30",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A sector is generically dependent continuant that is a subdivision of a system."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/30
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/477",
         rdfs:label "sector"
     
     SubClassOf: 
```

Comment:
Seems like you didn't change the classification?

Reply:
I fixed it.